intersection_transit_maneuvering/src/intersection_transit_maneuvering.cpp

```diff
@@ -123,6 +127,9 @@ namespace intersection_transit_maneuvering
                 new_maneuver.lane_following_maneuver.end_speed = maneuver.intersection_transit_straight_maneuver.end_speed;
                 new_maneuver.lane_following_maneuver.end_time = maneuver.intersection_transit_straight_maneuver.end_time;
 
+                new_maneuver.lane_following_maneuver.parameters.bool_valued_meta_data.clear();
+                new_maneuver.lane_following_maneuver.parameters.float_valued_meta_data.clear();
```

Comment:
Since you made a new message, the vectors will already be empty. No need to clear them unless you copied the data over like you were doing before.